Formula/grumpy.rb

```diff
@@ -0,0 +1,25 @@
+class Grumpy < Formula
+  desc "Python to Go source code transcompiler and runtime"
+  homepage "https://github.com/google/grumpy"
+  head "https://github.com/google/grumpy.git"
+
+  depends_on "go" => :build
```

Comment:
Is go a build-only dependency, or do you actually need go installed to run the application too?